test/e2e/wmco_test.go

```diff
@@ -33,12 +36,17 @@ func TestWMCO(t *testing.T) {
 	gc.numberOfNodes = int32(numberOfNodes)
 	gc.skipNodeDeletion = skipNodeDeletion
 	gc.sshKeyPair = sshKeyPair
+	require.NotEmpty(t, privateKeyPath, "private-key-path is not set")
+	gc.privateKeyPath = privateKeyPath
 	// When the OPENSHIFT_CI env var is set to true, the test is running within CI
 	if inCI := os.Getenv("OPENSHIFT_CI"); inCI == "true" {
 		// In the CI container the WMCO binary will be found here
 		wmcoPath = "/usr/local/bin/windows-machine-config-operator"
 	}
 
+	// test that the operator can deploy without the secret already created, we can later use a secret created by the
+	// individual test suites after the operator is running
+	t.Run("operator deployed without private key secret", testOperatorDeployed)
```

Comment:
Nit: I am not a fan of these long test names. But I can't think of a better name :-)